Packs/CortexAttackSurfaceManagement/Scripts/InferWhetherServiceIsDev/InferWhetherServiceIsDev_test.py

```diff
@@ -0,0 +1,39 @@
+import demistomock as demisto  # noqa: F401
+
+
+def test_canonicalize():
```

Comment:
Can you add docstrings similar to https://github.com/demisto/content/blob/99499d0afb3b431aba9d09b59b2d156e11731df9/Packs/CortexAttackSurfaceManagement/Integrations/CortexAttackSurfaceManagement/CortexAttackSurfaceManagement_test.py#L7, please?

Reply:
done